src/pymongo_migrate/mongo_migrate.py

```diff
@@ -32,6 +33,38 @@ def _deserialize(data, cls):
     return cls(**data)
 
 
+class MeasureTime:
```

Comment:
please change it to `_MeasureTime` since like you did originally - we don't want pymongo-migrate users to think this is API for public use

Reply:
Fixed.